.github/workflows/deploy.yml

```diff
@@ -32,23 +29,33 @@ jobs:
 
       - name: Build image and copy to local Docker registry
         run: |
-          nix develop --command std //marlowe-playground/oci-images/${{ matrix.image }}:load
+          nix build .#oci-images.x86_64-linux.all.copyToDockerDaemon
```

Comment:
@zeme-wana Can we have something in the backlog to clean up/unify image creation/publication? I don't think the playground devs could maintain this file.

Reply:
Sure I'll add a Story